docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -0,0 +1,333 @@
+
+#### Creating a SOA Suite Docker image
+
+You can also create a Docker image containing the SOA Suite binaries.
+A [sample](https://github.com/oracle/docker-images/tree/master/OracleSOASuite)
+is provided in the Oracle GitHub account that demonstrates how to create a Docker image
+to run SOA Suite.  
+
+Please consult the [README](https://github.com/oracle/docker-images/blob/master/OracleSOASuite/dockerfiles/README.md) file associated with this sample for important prerequisite steps,
+such as building or pulling the Server JRE Docker image, Oracle FMW Infrastructure Docker image, and downloading the SOA Suite installer binary.
+
+
+You must also install the [required patch]({{< relref "/userguide/introduction/introduction/_index.md#prerequisites" >}})
+to use this image with the operator.  A [sample](https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure/samples/12213-patch-fmw-for-k8s)
+is provided that demonstrates how to create a Docker image with the necessary patch installed. Use this patched image for building the SOA Suite image.
+
+Follow these steps to build the patched SOA image:
+
+After downloading the patch from My Oracle Support, you create the patched image
+by running the provided script:
+
+```bash
+cd docker-images/OracleFMWInfrastructure/samples/12213-patch-fmw-for-k8s
+./build.sh
+```
+
+This will produce an image named `oracle/fmw-infrastructure:12213-update-k8s`.
+
+All samples and instructions reference the pre-built and already patched image, `container-registry.oracle.com/middleware/fmw_infrastructure:12.2.1.3`. When building your own image, you will need to rename `oracle/fmw-infrastructure:12213-update-k8s` to `container-registry.oracle.com/middleware/fmw_infrastructure:12.2.1.3`.
+
+```
+$ docker tag oracle/fmw-infrastructure:12213-update-k8s container-registry.oracle.com/middleware/fmw_infrastructure:12.2.1.3
+```
+
+After cloning the repository and downloading the installer from Oracle Technology Network
+or e-delivery, you create your image by running the provided script:
+
+```bash
+cd docker-images/OracleFMWInfrastructure/dockerfiles
+./buildDockerImage.sh -v 12.2.1.3 -s
+```
+
+The image produced will be named `middleware/soasuite/oracle/soasuite:12.2.1.3`.
+
+The Oracle SOA Suite image created through the above step needs to be retagged
+from `middleware/soasuite/oracle/soasuite:12.2.1.3` to `oracle/soa:12.2.1.3` before continuing with the next steps.
+
+```bash
+$ docker tag middleware/soasuite/oracle/soasuite:12.2.1.3 oracle/soa:12.2.1.3
+```
+
+You can use this image to run the Repository Creation Utility and to run your domain using the “domain on a persistent volume” model.
+
+Before creating a domain, you will need to set up the necessary schemas in your database.
+
+#### Configuring access to your database
+
+SOA Suite domains require a database with the necessary schemas installed in them.
+A utility called Repository Creation Utility (RCU) is provided which allows you to create
```

Comment:
Please rephrase this line
"A utility called Repository Creation Utility (RCU) is provided which allows you to create"
to 
"The Repository Creation Utility (RCU) allows you to create"